exporter/awsxrayexporter/translator/sql_test.go

```diff
@@ -45,3 +50,30 @@ func TestClientSpanWithStatementAttribute(t *testing.T) {
 	testWriters.release(w)
 	assert.True(t, strings.Contains(jsonStr, "mysql://db.example.com:3306/customers"))
 }
+
+func TestClientSpanWithoutDBurlAttribute(t *testing.T) {
+	attributes := make(map[string]string)
+	attributes[semconventions.AttributeComponent] = "db"
+	attributes[semconventions.AttributeDBType] = "sql"
+	attributes[semconventions.AttributeDBInstance] = "customers"
+	attributes[semconventions.AttributeDBStatement] = "SELECT * FROM user WHERE user_id = ?"
+	attributes[semconventions.AttributeDBUser] = "readonly_user"
+	attributes[semconventions.AttributeDBURL] = ""
+	attributes[semconventions.AttributeNetPeerName] = "db.example.com"
+	attributes[semconventions.AttributeNetPeerPort] = "3306"
+	filtered, sqlData := makeSQL(attributes)
+	assert.NotNil(t, filtered)
+	assert.Equal(t, sqlData, &SQLData{
+		URL:            "localhost/customers",
+		DatabaseType:   "sql",
+		User:           "readonly_user",
+		SanitizedQuery: "SELECT * FROM user WHERE user_id = ?"})
+
+	w := testWriters.borrow()
```

Comment:
We want to remove the json assertions from this package since it happens at a different layer. For now let's make sure to leave out from new tests